src/http/client.c

```diff
@@ -1260,6 +1260,60 @@ int http_client_disable_verify_server(struct http_cli *cli)
 
 	return 0;
 }
+
+
+/**
+ * Change used certificate+key of TLS connection
+ *
+ * @param cli       HTTP Client
+ * @param chain     Cert (chain) + Key (PEM format)
+ * @param len_chain Length of certificate + key PEM string
+ *
+ * @return int 0 if success, otherwise errorcode
+ */
+int http_client_use_chainpem(struct http_cli *cli, const char *chain,
+			     int len_chain)
+{
+	int err;
+
+	if (!cli || !cli->tls)
+		return EINVAL;
+
+	err = tls_set_certificate_chain_pem(cli->tls, chain, len_chain);
```

Comment:
use C99 style declare and assign here ... `int err = ...`

Reply:
Done.